GiniVision/Classes/Core/Screens/Digital Invoice/DigitalLineItemTableViewCell.swift

```diff
@@ -207,7 +207,7 @@ class DigitalLineItemTableViewCell: UITableViewCell {
     }
     
     private func setup() {
-                
+        backgroundColor = UIColor.from(giniColor: viewModel?.giniConfiguration.digitalInvoiceBackgroundColor ?? GiniConfiguration.shared.digitalInvoiceBackgroundColor)
```

Comment:
Below on line 236 you used `digitalInvoiceLineItemsBackgroundColor` instead. Is this intentional?

Reply:
Oh, I see, this is for the view background color and on line 236 it's for the layer.